.editorconfig

```diff
@@ -3,8 +3,17 @@ root = true
 
 ; Unix-style newlines
 [*]
+charset = utf-8
 end_of_line = LF
+insert_final_newline = true
+trim_trailing_whitespace = true
 
-[*.php]
+[*.{php, html, twig}]
```

Comment:
sorry one last change: I think the space after commas should be removed - at least that's required for Unix glob patterns